packages/lib/server/repository/membership.ts

```diff
@@ -132,6 +132,82 @@ export class MembershipRepository {
     });
   }
 
+  static async findAllByUpIdIncludeMinimalEventTypes(
```

Comment:
just to fetch membership data and only fetch event types when it's non org workflow as we display team names and not event types in case of org.